Locale/de_DE/translations.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+return array(
+"Plugins" => "Erweiterungen",
```

Comment:
@cptsanifair shall I add this file?

Reply:
The updated strings are here:
https://github.com/aljawaid/PluginManager/blob/master/Locale/translations-starter-template.php